Sources/Support/DebugUI/InvalidProductErrorPayload+StatusMessage.swift

```diff
@@ -0,0 +1,45 @@
+//
+//  Copyright RevenueCat Inc. All Rights Reserved.
+//
+//  Licensed under the MIT License (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      https://opensource.org/licenses/MIT
+//
+//  InvalidProductErrorPayload+StatusMessage.swift
+//
+//  Created by Pol Piella on 4/10/25.
+
+import Foundation
+
+extension PurchasesDiagnostics.InvalidProductErrorPayload {
+    var statusMessage: String {
+        switch self.status {
+        case .valid:
+            return "Available for production purchases."
+        case .couldNotCheck:
+            return self.description
+        case .notFound:
+            return """
+                Product not found in App Store Connect. You need to create a product with identifier: \
+                '\(self.identifier)' in App Store Connect to use it for production purchases.
+                """
+        case .actionInProgress:
+            return """
+                Some process is ongoing and needs to be completed before using this product in production purchases, \
+                controlled either by the developer or by Apple (state: \(self.description))
```

Comment:
this is the only one that doesn't quite have remediation steps, what do I do if I run into this?